firmware/library/L1_Drivers/ST7066U.hpp

```diff
@@ -0,0 +1,196 @@
+
+// The file contains the driver class for the ST7066U Parallel LCD Driver.
+//
+// Usage:
+//      Initialize the driver by calling the constructor:
+//       ST7066U(BusMode busMode, DisplayLines numLines,
+//               Font fontStyle, ControlPins_t *pins)
+//      If the desired busMode is 4-bits, then initialize D3, D2, D1,
+//      and D0 in ControlPins_t as NULL.
+//      Characters can be generated on the display using the function:
+//          void displayText(const char *text, CursorPosition_t pos)
+#pragma once
+
+#include "L0_LowLevel/LPC40xx.h"
+#include "L1_Drivers/gpio.hpp"
+
+class ST7066UInterface
```

Comment:
Please follow Google C++ Style guide for naming variables, methods and classes.
https://google.github.io/styleguide/cppguide.html#General_Naming_Rules